src/coreclr/src/jit/lclvars.cpp

```diff
@@ -2821,14 +2823,15 @@ void Compiler::makeExtraStructQueries(CORINFO_CLASS_HANDLE structHandle, int lev
 
 void Compiler::lvaSetStructUsedAsVarArg(unsigned varNum)
 {
-#ifdef FEATURE_HFA
+    if (GlobalJitOptions::compFeatureHfa)
```

Comment:
I believe this if can be removed. The existing #ifdef for FEATURE_HFA was imo redundant.